docs/index.html

```diff
@@ -0,0 +1,154 @@
+<!doctype html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>PayPal JS SDK Documentation</title>
+        <style>
```

Comment:
Should this be using the paypal-fonts.css file instead of inlining these styles?

Reply:
Yes, good catch. Done! https://github.com/paypal/paypal-js/pull/698/commits/f4dc24d0d3d9d659eea8ac0946e35dd5265ef16a